include/deal.II/matrix_free/fe_evaluation.h

```diff
@@ -3580,17 +3724,29 @@ namespace internal
              "compatible vector."));
   }
 
-  // A class to use the same code to read from and write to vector
+
+
+  // Below, three classes (VectorReader, VectorSetter,
+  // VectorDistributorLocalToGlobal) implement the same interface and can be
+  // used to to read from vector, set elements of a vector and add to elements
+  // of the vector.
+  // FIXME: document interace
```

Comment:
I guess the current basic description is already better what we had before, so I suggest we skip the FIXME (we can always expand on this documentation - and as far as I remember, this is not part of the doxygen interface).

Reply:
> so I suggest we skip the FIXME

ok, will do.

Reply:
> as far as I remember, this is not part of the doxygen interface

true.

Reply:
@kronbichler fixed.